tensorboard/components/tensor_widget/colormap.ts

```diff
@@ -17,6 +17,21 @@ limitations under the License.
 
 const MAX_RGB = 255;
 
+/** Configuration for a colormap. */
+export interface ColorMapConfig {
+  /**
+   * Minimum value that the color map can map to without clipping.
+   * Must be a finite value.
+   */
+  min: number;
+
+  /**
+   * Minimum value that the color map can map to without clipping.
+   * Must be a finite value and be >= `min`.
```

Comment:
What happens in the case of equality?

Reply:
Then all finite values will mapped to a relative value of 0.5 and in turn be mapped to the mid-point of the colormap. Added comment here.